blueoil/networks/classification/quantize_example.py

```diff
@@ -0,0 +1,147 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.blocks import darknet
+from blueoil.networks.classification.base import Base
+from blueoil.layers import conv2d
+
+
+class FooNetwork(Base):
```

Comment:
IMO, `Foo` `Bar` is too casual to official sample code.
`SampleNetwork` is better.